datasets/indic_glue/indic_glue.py

```diff
@@ -0,0 +1,889 @@
+
+for lang in _CSQA_LANGS:
+    _NAMES.append(f"csqa.{lang}")
+
+for lang in _WSTP_LANGS:
+    _NAMES.append(f"wstp.{lang}")
+
+for lang in _iNLTKH_LANGS:
+    _NAMES.append(f"inltkh.{lang}")
+
+for lang in _BBCA_LANGS:
+    _NAMES.append(f"bbca.{lang}")
+
+for lang in _CVIT_MKB_CLSR:
+    _NAMES.append(f"cvit-mkb-clsr.{lang}")
+
+for lang in _IITP_MR_LANGS:
+    _NAMES.append(f"iitp-mr.{lang}")
+
+for lang in _IITP_PR_LANGS:
+    _NAMES.append(f"iitp-pr.{lang}")
+
+for lang in _ACTSA_LANGS:
+    _NAMES.append(f"actsa-sc.{lang}")
+
+for lang in _MD_LANGS:
+    _NAMES.append(f"md.{lang}")
+
+for lang in _WIKI_NER_LANGS:
+    _NAMES.append(f"wiki-ner.{lang}")
+
+
+class IndicGlueConfig(datasets.BuilderConfig):
+    """BuilderConfig for IndicGLUE."""
+
+    def __init__(self, data_url, citation, url, text_features, **kwargs):
+        """
+        Args:
+
+          data_url: `string`, url to download the zip file from.
+          citation: `string`, citation for the data set.
+          url: `string`, url for information about the data set.
+          text_features: `dict[string, string]`, map from the name of the feature
+        dict for each text field to the name of the column in the csv/json file
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(IndicGlueConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.data_url = data_url
+        self.citation = citation
+        self.url = url
+        self.text_features = text_features
+
+
+class IndicGlue(datasets.GeneratorBasedBuilder):
+
+    BUILDER_CONFIGS = [
+        IndicGlueConfig(
+            name=name,
+            description=_DESCRIPTIONS[name.split(".")[0]],
+            text_features=_TEXT_FEATURES[name.split(".")[0]],
+            data_url=_DATA_URLS[name.split(".")[0]],
+            citation=_CITATIONS[name.split(".")[0]],
+            url=_URLS[name.split(".")[0]],
+        )
+        for name in _NAMES
+    ]
+
+    def _info(self):
+        features = {text_feature: datasets.Value("string") for text_feature in six.iterkeys(self.config.text_features)}
+
+        if (
+            self.config.name.startswith("wnli")
+            or self.config.name.startswith("copa")
+            or self.config.name.startswith("actsa")
+        ):
+            features["label"] = datasets.Value("int32")
+
+        if self.config.name.startswith("csqa"):
+            features["options"] = datasets.features.Sequence(datasets.Value("string"))
+            features["out_of_context_options"] = datasets.features.Sequence(datasets.Value("string"))
+
+        if self.config.name.startswith("md"):
+            features["story_number"] = datasets.Value("int32")
+            features["id"] = datasets.Value("int32")
+
+        if self.config.name.startswith("wiki-ner"):
+            features["tokens"] = datasets.features.Sequence(datasets.Value("string"))
+            features["labels"] = datasets.features.Sequence(datasets.Value("string"))
```

Comment:
Could you name the labels `ner_tags` instead for consistency for other ner datasets ?
Also its feature type should be `datasets.Sequence(datasets.features.ClassLabel(...))`
see [germeval_14](https://github.com/huggingface/datasets/blob/master/datasets/germeval_14/germeval_14.py#L88-L118) for example

Reply:
Updated as requested